recipes/qh3/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "qh3" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/qh3-{{ version }}.tar.gz
+  sha256: eb527d8317746209509b9c575527577cdc9b3cfb0f49294fc1cd109b0570362c
+
+build:
+  skip: True  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  build:
+    - {{ stdlib('c') }}
+    - {{ compiler('c') }}
+    - {{ compiler('rust') }}
+    - cmake
+    - clang  # [linux]
+    - nasm  # [win]
+  host:
+    - python
+    - maturin >=1.2,<2.0
+    - pip
```

```suggestion
    - pip
    - clangdev
```
This is probably what you need

---

got it, let me try